update docs after legacy configs have been removed (#17634)
apache/druid: docs/querying/aggregations.md

| `finalize` | The finalize expression which can only refer to a single input variable, `o`. This expression is used to perform any final transformation of the output of the `fold` or `combine` expressions. If not set, then the value is not transformed. | No |
| `initialValue` | The initial value of the accumulator for the `fold` (and `combine`, if `InitialCombineValue` is null) expression. | Yes |
| `initialCombineValue` | The initial value of the accumulator for the `combine` expression. | No. Default `initialValue`. |

> | `isNullUnlessAggregated` | Indicates that the default output value should be `null` if the aggregator does not process any rows. If true, the value is `null`, if false, the result of running the expressions with initial values is used instead. | No. Defaults to `true`. |

Reviewer: ```suggestion
| `isNullUnlessAggregated` | Sets the default output value to `null` when the aggregator does not process any rows. If true, the value is `null`. If false, Druid computes the value as the result of running the expressions with initial values. | No. Defaults to `true`. |
```
nit awkward passive.
